Source/ResponseSerialization.swift

```diff
@@ -61,7 +61,7 @@ extension DataRequest {
                     let failureReason = "XML could not be serialized into response object: \(xml)"
                     let userInfo: [AnyHashable: Any] = [NSLocalizedFailureReasonErrorKey: failureReason]
                     let errorCode = AFError.responseSerializationFailed(reason: .inputDataNil)._code
-                    let error = NSError(domain: S3Error.Domain, code: errorCode, userInfo: userInfo)
+                    let error = NSError(domain: S3Error.Domain, code: errorCode, userInfo: userInfo as! [String : Any])
```

Comment:
Generally, using explicit casting (!) is a bad idea. Can't we just make ` userInfo` type `[String: Any]` on line 62?

Reply:
Are you looking at an older commit here? Cause when I opened the file to make the change it was already done.

Reply:
This was fixed in commit 397a93c.